### DISCUSSION TITLE
Uglifier parsing error when `'` found in front matter text

### DISCUSSION THREAD

**Discussion by garogarabed12** [alshedivat/al-folio#2756] [Q&A] [answered]
Hello,

When configuring the title attribute in the front matter with a text that includes a single quote `'`, for example in `announcement_2.md`:

`layout: post`
`title: I'm excited for this announcement`
`date: 2015-11-07 16:11:00-0400`
`inline: false`
`related_posts: false`
`----`

I'm getting an uglifier error of an unexpected token of whatever comes after the `'`, in this case: `m excited for this announcement`.

I have tried using the known escape characters, writing the title in double quotes, using `|` and `>`, but none of these approaches have worked so far. What am I missing here?

Thanks.

**Comment by riasat-sheikh** [comment]
Have you tried putting the whole string inside a quotation like, `title: "I'm excited for this announcement"`, Maybe? 
I can't reproduce the error, I have recently updated my website with the latest commit of al-folio

**Reply by george-gca** [reply]
I can't reproduce it either. Are you using the latest version?

**Reply by garogarabed12** [reply]
Yes I have. The quotation marks are then getting replaced by single quotations, bringing me back the starting point, i.e. the `m excited for this announcement` part getting cut off and interpreted as an unexpected token.

**Reply by george-gca** [reply]
Are you using the latest version? If not, try to check the difference between your local `_includes/news.liquid` file and [ours](https://github.com/alshedivat/al-folio/blob/master/_includes/news.liquid).

**Reply by garogarabed12** [reply]
@george-gca Yes I am, I forked the repository yesterday. And the diff is showing nothing, I have the same version.

**Reply by george-gca** [reply]
I can't reproduce it

![image](https://github.com/user-attachments/assets/ae6d35ec-9405-4126-85aa-bbafff19fbf7)

Are you using the correct quote symbol? `'`?

**Reply by garogarabed12** [reply]
@george-gca Yes, I believe so.

Well, I should've been clearer: it's crashing while deploying. Here's the log:

![image](https://github.com/user-attachments/assets/29e7c2f6-f052-4c6f-aa19-b13ecee38d4a)

![image](https://github.com/user-attachments/assets/9c3b08f7-64b9-4ac7-85cd-d56a04189dcc)

And this is the `.md` file:

![image](https://github.com/user-attachments/assets/d57b25b9-be2e-4f10-be39-abb975e97d75)

**Reply by george-gca** [reply, marked as answer]
The prints are rather useless, it would be easier if you could paste the full error trace, or even share with us your repo.

Giving that what is causing this issue is really the quotations, I can only think that maybe one file that is processed before is "opening" some quotes that are not closed, and it is looking for the "closure".

**Reply by garogarabed12** [reply]
Hello @george-gca!

Apologies for the late reply, and thank you for your help. After re-cloning and setting up the repository again, the issue no longer occurs. Not sure what exactly fixed it, but it seems resolved now

Regards,
Garo
